src/brogue/Rogue.h

```diff
@@ -2329,6 +2330,11 @@ typedef struct creatureIterator {
 
 enum NGCommands {
     NG_NOTHING = 0,
+    NG_FLYOUT_PLAY,
+    NG_FLYOUT_VIEW,
+    NG_FLYOUT_OPTIONS,
+    NG_GAME_VARIANT,
+    NG_GAME_MODE,
```

Comment:
We are adding more functions for `rogue.nextGame`. It might be worth updating the comment Rogue.h:2494 as a minimum.
Personally I don't love the overloading of `nextGame` to mean `nextMenuAction`, but I don't like `nextGame` much anyway (since it stops semantically meaning `nextGame` once the game starts).
So no required change from me, but you could consider using a different file-scoping variable to manage flyout changes and menu select state and then populating `nextGame` only once the menu interactions are over. Optionally.

Reply:
I actually started with a different file-scoping variable, but didn't like all the connecting code. I came to think of `nextGame` as a misnomer. It already includes actions that don't involve starting a game (view recording, high scores, quit), so to me it already means `nextMenuAction`, despite the name.